spec/unit/libraries/unit_converter_spec.rb

```diff
@@ -0,0 +1,92 @@
+# coding: UTF-8
+require_relative '../spec_helper'
+require 'splunk_app'
+
+describe '#unit_converter' do
+  subject { CernerSplunk.convert_to_bytes(size) }
+
+  context 'when size specified in TB' do
+    let(:size) { '2TB' }
+    it { is_expected.to eq(2_199_023_255_552) } # underscore because rubocop was complaining.
```

Comment:
Yes rubocop can be a real pain in the [cloud](https://www.flickr.com/groups/cloud-to-butt/pool/) sometimes... no need to comment because if someone changes it rubocop will yell again... or it'll stop yelling because the rules changed again.